src/cow-react/modules/swap/pure/EthFlow/EthFlowBanner/index.tsx

```diff
@@ -0,0 +1,145 @@
+import { Separator } from 'theme'
+import styled from 'styled-components/macro'
+import { ChevronUp, ChevronDown } from 'react-feather'
+import { Currency, Token } from '@uniswap/sdk-core'
+
+import { ETH_FLOW_SLIPPAGE } from '@cow/modules/swap/state/EthFlow/updater'
+import { PERCENTAGE_PRECISION } from 'constants/index'
+import { EthFlowBannerCallbacks } from '@cow/modules/swap/containers/EthFlow/EthFlowBanner'
+import ethFlowIcon from 'assets/svg/ethFlow.svg'
+import { darken } from 'polished'
+
+const BannerWrapper = styled.div`
```

Comment:
Let's move styled elements to `styled.tsx` file and import as `import * as styledEl from './styled'`

Reply:
nice, hyeah good call